src/Microsoft.Health.Dicom.Api.UnitTests/Features/Filters/AcceptContentFilterAttributeTests.cs

```diff
@@ -130,7 +130,11 @@ public void GivenARequestWithNoAcceptHeader_WhenValidatingTheContentTypeAgainstS
     [InlineData("multipart/related; type=\"application/octet_stream\"", (int)HttpStatusCode.NotAcceptable)]
     [InlineData("multipart/related; type=\"application/octetStream\"", (int)HttpStatusCode.NotAcceptable)]
     [InlineData("multipart/related; type=\"application/dicom\"", null)]
+    [InlineData("multipart/related; type='application/dicom'", (int)HttpStatusCode.NotAcceptable)]
+    [InlineData("multipart/related; type=application/dicom", null)]
     [InlineData("multipart/relATed; type=\"applICation/DICom\"", null)]
+    [InlineData("multipart/relATed; type='applICation/DICom'", (int)HttpStatusCode.NotAcceptable)]
+    [InlineData("multipart/relATed; type=applICation/DICom", null)]
```

Comment:
We should add a test case with two parameters, like: `multipart/related; type=application/dicom; transfer-syntax="*"`